py/desiutil/annotate.py

```diff
@@ -0,0 +1,376 @@
+                    izero = h.header.index('BZERO')
+                    if izero > iscale:
+                        hc.header.insert(iscale - 1, ('BSCALE', h.header['BSCALE'], h.header.comments['BSCALE']), after=True)
+                        hc.header.insert(iscale, ('BZERO', h.header['BZERO'], h.header.comments['BZERO']), after=True)
+                    else:
+                        hc.header.insert(izero - 1, ('BZERO', h.header['BZERO'], h.header.comments['BZERO']), after=True)
+                        hc.header.insert(izero, ('BSCALE', h.header['BSCALE'], h.header.comments['BSCALE']), after=True)
+            new_hdus.append(hc)
+    new_hdulist = fits.HDUList(new_hdus)
+    new_hdulist._open_kwargs = kwargs
+    log.debug(new_hdulist._open_kwargs)
+    try:
+        ext = int(extension)
+    except ValueError:
+        ext = extension
+    try:
+        hdu = new_hdulist[ext]
+    except (IndexError, KeyError):
+        raise
+    return new_hdulist
+
+
+def _options():
+    """Parse command-line options.
+    """
+    parser = ArgumentParser(description="Add units or comments to a FITS file.",
+                            prog=os.path.basename(sys.argv[0]))
+    parser.add_argument('-c', '--comments', action='store', dest='comments', metavar='COMMENTS',
+                        help="COMMENTS should have the form COLUMN='comment':COLUMN='comment'.")
+    parser.add_argument('-C', '--csv', action='store', dest='csv', metavar='CSV',
+                        help="Read annotations from CSV file.")
+    parser.add_argument('-e', '--extension', dest='extension', action='store', metavar='EXT', default='1',
+                        help="Update FITS extension EXT, which can be a number or an EXTNAME. If not specified, HDU 1 will be updated, which is standard for simple binary tables.")
+    parser.add_argument('-o', '--overwrite', dest='overwrite', action='store_true',
+                        help='Overwrite the input FITS file.')
+    parser.add_argument('-t', '--test', dest='test', action='store_true',
+                        help='Test mode; show what would be done but do not change any files.')
+    parser.add_argument('-u', '--units', action='store', dest='units', metavar='UNITS',
+                        help="UNITS should have the form COLUMN='unit':COLUMN='unit'.")
+    parser.add_argument('-v', '--verbose', dest='verbose', action='store_true',
+                        help='Print extra debugging information.')
+    parser.add_argument('-V', '--version', action='version', version='%(prog)s ' + desiutilVersion)
+    parser.add_argument('-Y', '--yaml', action='store', dest='yaml', metavar='YAML',
+                        help="Read annotations from YAML file.")
+    parser.add_argument('fits', metavar='FITS', help='FITS file to modify.')
+    parser.add_argument('output', metavar='FITS', nargs='?',
+                        help='Write to new FITS file. If --overwrite is specified, this value is ignored.')
```

Comment:
I'm not sure exactly how metavar is used, but let's give them more descriptive names if possible so that the short form of the help is more understandable, e.g.
```
usage: annotate_fits [-h] [-c COMMENTS] ... INPUT_FITS [OUTPUT_FITS]
```

Reply:
Well, that example is *exactly* how `metavar` is used, and that's reasonable.

Reply:
Done in [dab51a4](https://github.com/desihub/desiutil/pull/199/commits/dab51a474c40e19cc3dbbc9c2b8cfd0d4d4e199b).